beacon-chain/validator-monitor/process_attestation.go

```diff
@@ -0,0 +1,165 @@
+package validator_monitor
+
+import (
+	"fmt"
+
+	types "github.com/prysmaticlabs/eth2-types"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/config/params"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/attestation"
+	"github.com/prysmaticlabs/prysm/runtime/version"
+	"github.com/prysmaticlabs/prysm/time/slots"
+	"github.com/sirupsen/logrus"
+)
+
+// processAttestation logs in the event that one of our tracked validators'
+// appears in the attesting indices and this attestation was not included
+// before.
+func (s *Service) processAttestation(state state.BeaconState, att *ethpb.Attestation, included bool) {
```

Comment:
We are anti passing boolean flag in function, especially in this incident where it's only for additional log. There's better approach for that if we do insist for additional `logMessage`

```suggestion
func (s *Service) processAttestation(state state.BeaconState, att *ethpb.Attestation) {
```